styles/prosilver/theme/profile_side_switcher.css

```diff
@@ -20,7 +20,7 @@
 	background-image: none;
 }
 @media only screen and (max-width: 700px), only screen and (max-device-width: 700px) {
-	.leftsided div.postbody {
+	.section-viewtopic .leftsided div.postbody {
```

Comment:
hI @lavigor,

you have revert this fix: https://github.com/Tatiana5/profile_side_switcher/issues/9 could you fix with an other way please ?